pkg/installation/move.go

```diff
@@ -201,13 +201,14 @@ func moveOrCopyDir(from, to string) error {
 	if le, ok := err.(*os.LinkError); err != nil && ok {
 		if errno, ok := le.Err.(syscall.Errno); ok && errno == 18 {
 			glog.V(4).Infof("Cross-device link error (ERRNO=18), fallback to manual copy")
-			return copyDir(from, to)
+			return copy(from, to)
```

Comment:
What is our stand overriding built-ins?

Reply:
I would like to avoid that.

Reply:
Go is surprisingly forgiving when it comes to these things.

I will rename this to `copyTree`.